pkg/web/provider.go

```diff
@@ -61,6 +62,9 @@ func onStartup(event interface{}) error {
 	// health checks
 	r.GET("/status", checker.NewOverviewHandler(repo))
 	r.GET("/status/{name}", checker.NewStatusHandler(repo))
+	if e.Config.Prometheus.Enabled {
```

Comment:
This can be moved to the plugin. The `plugin.OnStartup` event contains the Global configurations for Janus and this event runs when Janus starts. On the `plugin.OnAdminAPIStartup` you have access to the admin router to add this route.